course3/homework-string/my_string.cc

```diff
@@ -1,46 +1,58 @@
+#include "stdafx.h"
 #include "my_string.h"
 
 #include <cstring>
 
-namespace Maciek{
-	
-String::String(const char *inchar)
-    :mBuffer(new char[std::strlen(inchar) + 1])
-{
-	std::strcpy(mBuffer.get(), inchar);
-}
-
-String::String(const String& str)
-{
-	if (!str.empty())
+namespace Maciek {
+
+	String::String(): mBuffer(nullptr), mSize(0) { }
```

Comment:
no need to initialize mBuffer. It will be set to null in its default constructor.